sources/EXTERNALFORMAT.TXT

```diff
@@ -0,0 +1,70 @@
+New architecture for character input-output and alternative external formats
```

Comment:
we really need to put together a Medley 4 release manual, formatted and fixed up